### ISSUE TITLE
with eos data, interface assertion fails with 'No LLDP peering info' on all interfaces

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#161]
**Describe the bug**
LLDP is in this database
```
(suzieq-ITL68y-d-py3.7) jpiet@a2:~/suzieq$ python3 suzieq/cli/suzieq-cli -c ~/configs/eos.yml interface assert
    namespace   hostname ifname  ...               timestamp          assertReason  assert
0         eos     edge01   eth0  ... 2020-06-17 03:04:53.822  No LLDP peering info    fail
3         eos     edge01   eth1  ... 2020-06-17 03:04:53.822  No LLDP peering info    fail
6         eos     edge01   eth2  ... 2020-06-17 03:04:53.822  No LLDP peering info    fail
30        eos   internet   eth0  ... 2020-06-17 03:04:53.720  No LLDP peering info    fail
32        eos   internet   swp1  ... 2020-06-17 03:04:53.720  No LLDP peering info    fail
33        eos   internet   swp2  ... 2020-06-17 03:04:53.720  No LLDP peering info    fail
91        eos  server101   eth0  ... 2020-06-17 03:04:54.219  No LLDP peering info    fail
96        eos  server102   eth0  ... 2020-06-17 03:04:53.643  No LLDP peering info    fail
101       eos  server103   eth0  ... 2020-06-17 03:04:53.992  No LLDP peering info    fail
106       eos  server104   eth0  ... 2020-06-17 03:04:53.957  No LLDP peering info    fail

[10 rows x 9 columns]
Assert failed
```

```
(suzieq-ITL68y-d-py3.7) jpiet@a2:~/suzieq$ python3 suzieq/cli/suzieq-cli -c ~/configs/eos.yml lldp summarize
                     eos
deviceCnt             10
nbrCnt                60
peerHostnameCnt       60
uniquePeerMgmtIPCnt   12
missingPeerInfoCnt    18
```

**Comment by ddutt** [comment]
But there's no LLDP peering info for those interfaces.

**Comment by ddutt** [comment]
Fixed with 3a3c3f4ea5c83aa8eb9ea65185be344fbd2ffc86
